### PR TITLE
Upgrade to 5.5.0.1

### PR DESCRIPTION
**Pull request by snaggen**
The version of 5.5.0.1 works again. I updated the version tag to stable, since this is probably as stable as Skype gets on linux.

### CODE REVIEW

**Comment by alexlarsson** [comment]
I'd like to merge this to flathub, is that ok with you?

File: com.skype.Client.json [hunk 1]
```diff
@@ -15,16 +15,18 @@
...
         /* Network access */
         "--share=network",
-        /* File system access */
-        "--filesystem=xdg-download",
-        "--filesystem=home:ro",
+  	   /* File system access */
+	   "--filesystem=home:ro",
+	   "--filesystem=xdg-download",
+	   "--filesystem=xdg-config",
+	   "--filesystem=xdg-cache",
```
Comment (alexlarsson) [inline com.skype.Client.json]: Why do you add xdg-config and xdg-cache? I don't see any reason to expose the host versions of these?

Comment (snaggen) [reply com.skype.Client.json]: No, they are just left-overs since my desperate debuging of the previous failing version.

**Comment by snaggen** [comment]
And merging to flathub is fine with me.

**Comment by alexlarsson** [comment]
Its on flathub now

**Pull request closed without merging**